numpy/typing/tests/data/pass/shape.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, cast, Literal, NewType, TypeVar
```

Comment:
These tests look more like they belong in `numpy/typing/test/data/reveal`.

And because those are `.pyi` tests, the runtime code doesn't matter anymore